src/background/services/globalConfigService.ts

```diff
@@ -38,7 +39,9 @@ const EXPORTABLE_PROPERTIES = [
   'dynamicAdapter',
   'preferedOverlayStorage',
   'myDapplets',
-  'connectedAccountContractAddress',
+  'connectedAccountsTestnetContractAddress',
+  'connectedAccountsMainnetContractAddress',
```

Comment:
These hardcoded networks must be fixed in DAP-1552

Reply:
add comment in https://dapplets-org.atlassian.net/browse/DAP-1552